### PR TITLE
chore: eslint migration

### PR DESCRIPTION
**Pull request by Averethel**

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  extends: ["@carforyou/eslint-config"],
```
Comment (masone) [inline .eslintrc.js:2]: I interpret that this extends all rules we have, but it's probably just the base ones. Not sure.. might be just me stumbling here

File: package.json [hunk 2]
```diff
@@ -15,15 +15,17 @@
   "author": "CAR FOR YOU",
   "homepage": "https://github.com/autoricardo/carforyou-phraseapp-pkg#readme",
   "devDependencies": {
-    "@carforyou/tslint-rules": "3.0.0",
+    "@carforyou/eslint-config": "1.1.0-initial-rules-ca8a1992a789bb4c88f8f9a642c9c867859b13b9.7",
```
Comment (masone) [inline package.json:18]: want Renovate to take care of this or change before merging?

File: package.json [hunk 3]
```diff
@@ -15,15 +15,17 @@
...
     "@pika/plugin-build-node": "^0.3.16",
     "@pika/plugin-copy-assets": "^0.3.16",
     "@pika/plugin-simple-bin": "^0.3.16",
     "@pika/plugin-standard-pkg": "^0.3.16",
+    "eslint": "^6.8.0",
+    "eslint-plugin-prettier": "^3.1.3",
     "prettier": "^2.0.5",
     "semantic-release": "^17.0.3",
-    "tslint": "^5.20.1",
+    "tslint": "^6.1.2",
```
Comment (masone) [inline package.json:28]: you probably don't need this anymore

File: src/push.js [hunk 4]
```diff
@@ -16,7 +15,7 @@ export default () => {
   const pushOutput = execSync(
     `${bin} push --access-token $PHRASEAPP_TOKEN --wait`
   ).toString()
-  console.log(pushOutput)
+  console.info(pushOutput)
```
Comment (masone) [inline src/push.js:18]: well, that's an interesting way to get around `no-console` :D

Comment (lkappeler) [reply src/push.js:18]: it's actually configured to allow info, warning and error in the linter package

File: .eslintrc.js (hunk 1 above)
Comment (Averethel) [reply .eslintrc.js:2]: It's the base javascript/typescript set of rules that is the default. Nothing that the docs can't fix I'd say.

**Pull request merged**